### PR TITLE
Website revision

### PR DESCRIPTION
**Pull request by kuzdogan**
## Summary by CodeRabbit

- **New Features**
	- Introduced new components: `AboutSection`, `Contact`, `FAQ`, `SupportedChains`, and `Tooling`.
	- Added animations for various components using `framer-motion`.
	- Implemented a responsive design for the `Header` and `LandingPage` components.
  
- **Bug Fixes**
	- Updated tooltip implementation for consistency and improved functionality.

- **Style**
	- Enhanced visual presentation of buttons and select components.
	- Adjusted CSS for improved layout and responsiveness across components.

- **Chores**
	- Removed deprecated files and components to streamline the codebase.
	- Updated dependency versions for improved compatibility and features.
	- Added new custom animations and keyframes in the Tailwind configuration.

### CODE REVIEW

**Comment by kuzdogan** [comment]
Ok this is ready :) @manuelwedler Please have a look. You don't need to look into the code in detail IMO but now I realize it's not too complicated. After you give a go I'll merge into staging.

**Comment by kuzdogan** [comment]
@coderabbitai review

**Comment by kuzdogan** [comment]
@coderabbitai full review

File: src/pages/LandingPage/AboutSection.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                <p className="text-lg md:text-xl text-gray-600">
+                  Sourcify is and always will be 100% open source.{" "}
+                  <a
+                    className="text-ceruleanBlue-500 font-medium link-underline"
+                    href="https://github.com/ethereum/sourcify"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    Github →
+                  </a>
```
[the remark this answers is not in the document]
Comment (kuzdogan) [reply src/pages/LandingPage/AboutSection.tsx:39]: Fix all

File: src/pages/LandingPage/AboutSection.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+                  As a
+                  <a href="https://verifieralliance.org" target="_blank" rel="noopener noreferrer">
+                    <img src="/verifier-alliance.svg" alt="Verifier Alliance" className="h-16 inline-block -m-2" />
+                  </a>{" "}
+                  member, Sourcify fosters open standards, data sharing, interoperability, and collaboration between all
+                  source code verifiers.
+                </p>
+              </div>
+            </div>
+          </motion.div>
```
[the remark this answers is not in the document]
Comment (kuzdogan) [reply src/pages/LandingPage/AboutSection.tsx:106]: Do the fix

File: src/pages/LandingPage/SupportedChains.tsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+          </h1>
+        ) : (
+          <h1 className="text-3xl md:text-5xl text-ceruleanBlue-500 font-bold">Supported Chains</h1>
+        )}
+      </motion.div>
+
+      <Tooltip
+        className="!rounded-xl !text-lg !bg-ceruleanBlue-100 !text-ceruleanBlue-600 !opacity-100 !px-6 !py-4 !z-50"
+        id="supported-chains-tooltip"
+      />
```
[the remark this answers is not in the document]
Comment (kuzdogan) [reply src/pages/LandingPage/SupportedChains.tsx:63]: Do the fix

File: src/pages/LandingPage/Contact.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+            <img src={discord} alt="Discord logo" className="w-28 m-auto" style={{ height: "35px" }} />
+          </a>
+
+          <a
+            className="bg-gray-100 hover:bg-gray-200 rounded-xl font-bold p-4 w-64 text-center flex items-center justify-center transition duration-200"
+            href="mailto:hello@sourcify.dev"
+          >
+            hello@sourcify.dev
+          </a>
+        </div>
```
[the remark this answers is not in the document]
Comment (kuzdogan) [reply src/pages/LandingPage/Contact.tsx:36]: Do teh fix

**Review by manuelwedler — COMMENTED** [review]
Really great job here. Looks so much nicer now :)

I mostly reviewed visually. Some things I found:

- I saw the chain selector does not change the chain stats graph anymore. Was that intentional?
- I see this in the lookup page: image
- On the verify page, the arrow overflows the input: 
image
- On the verify page with a smaller screen width, the X in the menu can be cut off:
image
- barely noticeable, but the Documentation button is not centered on mobile:
image
- It bugs me a bit that the Verifier Alliance text is not on one line with the rest. I know it is a picture. Maybe the logo can just be put after the whole text?
image
- The footer seems weirdly arranged. The middle column is not in the center. But seems like it was like this before. 
image
- More subjectively, I found this view a bit unordered. It uses so many different font sizes. I think it makes the "Download..." link a bit lost. However, I don't have good proposal here other than using less different font sizes. 
image

File: src/pages/LandingPage/Tooling.tsx [hunk 5]
```diff
@@ -0,0 +1,310 @@
...
+            }}
+          >
+            {isCodeInView ? displayedCode.slice(hardhatExampleJS.length) : hardhatExampleBash}
+          </SyntaxHighlighter>
+        )}
+        {activeTab === "remix" && (
+          <div className="flex flex-col justify-center">
+            <div className="mb-4 text-center">
+              <a
+                href="https://remix.ethereum.org"
```
[the remark this answers is not in the document]
Comment (manuelwedler) [reply src/pages/LandingPage/Tooling.tsx:258]: Maybe there is even an action to focus it?

**Comment by kuzdogan** [comment]
> I saw the chain selector does not change the chain stats graph anymore. Was that intentional?

Yes I think it was looking a bit odd, it's much cleaner with a just sorted graph

**Comment by kuzdogan** [comment]
Incorporated the changes, please have a final short look 


https://deploy-preview-6--sourcify-ui.netlify.app/

**Review by manuelwedler — APPROVED** [review]
looks great!

**Pull request merged**